Marlin/src/module/probe.cpp

```diff
@@ -564,7 +564,7 @@ static float run_z_probe() {
     }
   #endif
 
-  return current_position[Z_AXIS] + zprobe_zoffset;
+  return current_position[Z_AXIS];
```

Comment:
Will removing `+ zprobe_zoffset` work for Cartesian machines?

Reply:
It is added in the probe_pt routine; so for Cartesians it won't make any difference.